CondTools/Geometry/test/calogeometry2026writer.py

```diff
@@ -1,8 +1,9 @@
 import FWCore.ParameterSet.Config as cms
 
-process = cms.Process("CaloGeometryWriter")
+from Configuration.Eras.Era_Phase2C17I13M9_cff import Phase2C17I13M9
```

Comment:
have you considered using the mechanism introduced at https://github.com/cms-sw/cmssw/pull/45764 to avoid having to always manually bump up the version here and elsewhere?